commands-yml/commands/device/clipboard/get-clipboard.yml

```diff
@@ -0,0 +1,78 @@
+---
+name: Get Clipboard
+short_description: Get the content of the system clipboard
+description:
+  >
+    Get the content of the system clipboard
+
+example_usage:
+  java:
+    |
+      driver.getClipboardText();
+  python:
+    |
+      self.driver.get_clipboard_text()
```

Comment:
`get(set)_clipboard_text` is just a specific case of the generic `get(set)_clipboard` call, which only works with plain text. The same in Java. There are also some specific methods in Java for setting images or URLs in iOS, but these generic ones exist everywhere